test/e2e/egress_test.go

```diff
@@ -825,3 +911,21 @@ func (data *TestData) waitForEgressRealized(egress *v1beta1.Egress) (*v1beta1.Eg
 	}
 	return egress, nil
 }
+
+func (data *TestData) deleteEgress(egressName string) error {
+	err := data.crdClient.CrdV1beta1().Egresses().Delete(context.TODO(), egressName, metav1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	err = wait.PollImmediate(200*time.Millisecond, waitEgressRealizedTimeout, func() (done bool, err error) {
+		_, err = data.crdClient.CrdV1beta1().Egresses().Get(context.TODO(), egressName, metav1.GetOptions{})
+		if err != nil && errors.IsNotFound(err) {
+			return true, nil
+		}
+		return false, nil
+	})
+	if err != nil {
+		return fmt.Errorf("wait for Egress %s to be deleted failed: %v", egressName, err)
+	}
+	return nil
```

Comment:
Egress doesn't have any finalizer, it will be deleted immediately after the `Delete` func returns, so no point to poll. That's why no delete func is extracted from other tests.

Reply:
Got you. Changed.